onadata/libs/serializers/submission_review_serializer.py

```diff
@@ -56,7 +61,21 @@ def create(self, validated_data):
                 note = Note.objects.create(**note_data)
                 validated_data['note'] = note
 
-        return SubmissionReview.objects.create(**validated_data)
+        submission_review = SubmissionReview.objects.create(**validated_data)
+
+        from multidb.pinning import use_master
+
+        with use_master:
+            message = {
+                'type': SUBMISSION_REVIEW,
+                'json': submission_review.instance.json
+            }
+            message = json.dumps(message)
```

Comment:
Can we skip converting the message into json here and just let the message be converted [here](https://github.com/onaio/onadata/pull/1793/files#diff-3f5ab3fb0914a3a9f192c4016ffe780bR38)  ? It would remove the need for the try/except I believe ?

Reply:
@DavisRayM it would be good to do that, but I intended to reuse the `MessageSerializer`'s  `create` function and this meant to have the message as a string because of this restriction in the message field.
`message = serializers.CharField(source='description', allow_blank=False)`.

Changing this to maybe `serializers.JSONField`could have an impact on how this serializer is used.

Reply:
Mhh okay